src/opera/test/util/test_input_validation.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python
+
+"""
+=================
+test_input_validation.py
+=================
+
+Unit tests for the util/input_validation.py module.
+"""
+import os
+import tempfile
+import unittest
+from os.path import abspath, exists, getsize
+
+from pkg_resources import resource_filename
+
+from opera.util.input_validation import validate_disp_inputs
+from opera.util.logger import PgeLogger
+
+class MockRunConfig:
+    """Mock runconfig for testing"""
+
+    def __init__(self, sas_config):
+        self._sas_config_dict = sas_config
+
+    @property
+    def sas_config(self):
+        return self._sas_config_dict
+
+class InputValidationTestCase(unittest.TestCase):
+    """Base test class using unittest"""
+
+    starting_dir = None
+    working_dir = None
+    test_dir = None
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        """
+        Set up directories for testing
+        Initialize other class variables
+
+        """
+        cls.starting_dir = abspath(os.curdir)
+        cls.test_dir = resource_filename(__name__, "")
+
+        os.chdir(cls.test_dir)
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        """
+        At completion re-establish starting directory
+        -------
+        """
+        os.chdir(cls.starting_dir)
+
+    def setUp(self) -> None:
+        """
+        Use the temporary directory as the working directory
+        -------
+        """
+        self.working_dir = tempfile.TemporaryDirectory(
+            prefix="test_input_validation_", suffix='_temp', dir=os.curdir
+        )
+        os.chdir(self.working_dir.name)
+
+    def tearDown(self) -> None:
+        """
+        Return to starting directory
+        -------
+        """
+        os.chdir(self.test_dir)
+        self.working_dir.cleanup()
+
+    def test_validate_disp_inputs(self):
```

Comment:
I appreciate the creation of the `test_input_validation.py` module, but I would prefer to see this test included in with the `test_disp_s1_pge.py` module.